raiden/ui/cli.py

```diff
@@ -106,6 +109,7 @@ def write_stack_trace(ex: Exception) -> None:
     )
     with file as traceback_file:
         traceback.print_exc(file=traceback_file)
+        traceback.print_exc()
```

Comment:
Do you really want to print it to both stdout and the traceback file?

Reply:
https://github.com/raiden-network/raiden/pull/6145#issuecomment-626739075